pkg/client/client.go

```diff
@@ -0,0 +1,70 @@
+package client
```

Comment:
@aclevername are you doing tests for this file?

Reply:
missed this... will do

Reply:
added what I could